flybywiresim/aircraft
# refactor: move auto baro unit to extras host and enable on a380x

Fixes #[issue_no]

## Summary of Changes

Move the auto baro unit code to the A32NX extras host and add it to the A380X extras host. Fix the default setting (AUTO) to match everywhere.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

In one of the planes:
- Set the baro unit to auto in the flypad.
- Spawn at an airport in the USA and ensure the unit auto-inits to in.Hg.
- Spawn in most of the rest of the world and ensure the unit auto-inits to hPa.
- Set the baro unit to hPa in the flypad, spawn in the USA, and ensure the unit auto-inits to hPa.
- Set the baro unit to in.Hg in the flypad, spawn somewhere living in the future, and ensure the unit auto-inits to in.Hg.

In the other:
- Repeat the first couple of steps with AUTO mode.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
